packages/components/src/components/UtilityCard/UtilityCard.tsx

```diff
@@ -9,8 +9,8 @@ export enum InteractiveElementTypeUtilityCard {
 }
 
 export type UtilityCardProps = {
-  /** Sets the card image source */
-  imageSrc: string;
+  /** Sets the category emoji */
```

Comment:
I wouldn't add the `category` here because this component needs to be generic. So, we could say: `Set an emoji for the card`. What do you think?